introduction.md

```diff
@@ -0,0 +1,3 @@
+# Shengli Li

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I like the training for GitHub very much!

Reply:
added my favorite Code School path in my introduction.